src/cmt_decode_msgpack.c

```diff
@@ -559,7 +544,7 @@ static int unpack_summary_quantiles(mpack_reader_t *reader, size_t index, void *
 
     decode_context = (struct cmt_msgpack_decode_context *) context;
 
-    return cmt_mpack_unpack_array(reader, unpack_summary_quantile, context);
```

Comment:
I was using context and not decode_context because context is already a `void *`, it's nothing in practical terms.

Reply:
so the right change is to remove `decode_context`